lib/paypal_express/api.rb

```diff
@@ -425,7 +425,7 @@ def authorize_or_purchase_payment(kb_account_id, kb_payment_id, kb_payment_trans
           end
 
           properties = merge_properties(properties, options)
-          dispatch_to_gateways(api_call_type, kb_account_id, kb_payment_id, kb_payment_transaction_id, kb_payment_method_id, amount, currency, properties, context, gateway_call_proc)
+          dispatch_to_gateways(api_call_type, kb_account_id, kb_payment_id, kb_payment_transaction_id, kb_payment_method_id, amount, currency, properties, context, gateway_call_proc, nil, {:payer_id => options[:payer_id]})
```

Comment:
Should we check if `payer_id` is `nil`? Otherwise, we might overwrite a real value returned by PayPal?